src/ontology/mondo-ingest.Makefile

```diff
@@ -63,8 +63,7 @@ $(TMPDIR)/ordo_relevant_signature.txt: component-download-ordo.owl | $(TMPDIR)
 #  - #60 needs to be fixed at source, but a workaround can probably be implemented in this goal
 $(COMPONENTSDIR)/omim.owl: $(TMPDIR)/omim_relevant_signature.txt | component-download-omim.owl
 	if [ $(COMP) = true ]; then $(ROBOT) remove -i $(TMPDIR)/component-download-omim.owl.owl --select imports \
-		rename --mappings config/property-map-1.sssom.tsv --allow-missing-entities true \
-		rename --mappings config/property-map-2.sssom.tsv --allow-missing-entities true \
+		rename --mappings config/property-map.sssom.tsv --allow-missing-entities true --allow-duplicates true \
```

Comment:
## Updated components goals
Updated all the components goals like this. Consolidated to just a single line, using the new filename and adding `--allow-duplicates true`.